Makefile

```
@@ -19,10 +21,16 @@ endif
 KONG_PACKAGE_NAME?="kong-community-edition"
 KONG_CONFLICTS?="kong-enterprise-edition"
 KONG_LICENSE?="ASL 2.0"
-KONG_SOURCE_LOCATION?="$$PWD/../kong/"
-KONG_VERSION?="0.0.0"
 PRIVATE_REPOSITORY?=true
 KONG_TEST_CONTAINER_NAME?=localhost:5000/kong
+KONG_SOURCE_LOCATION?="$$PWD/../kong/"
+
+KONG_VERSION?=`grep tag $(KONG_SOURCE_LOCATION)/kong-*.rockspec | awk -F" " '{print $$3}' | sed 's/"//g'`
```

the value of the `tag` field there is always the same as the `*` bit in `kong-*.rockspec`, but extracting it from the filename is more reliable than awk+sed'ding out of the contents.
```suggestion
KONG_VERSION?=`echo $(KONG_SOURCE_LOCATION)/kong-*.rockspec | sed 's,.*/,,' | cut -d- -f2`
```